dist/index.html

```diff
@@ -3,9 +3,34 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpack Exercise</title>
+    <title>TV shows</title>
+    <link href="https://fonts.googleapis.com/css2?family=Indie+Flower&family=Lobster&family=Poppins:wght@200;400;700&family=Shadows+Into+Light&display=swap" rel="stylesheet">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
 <script defer src="main.js"></script></head>
 <body>
-    <h1>Hello webpack!</h1>
+    <main>
+    <header>
+        <nav>
+            <ul class="header-list">
+                <li><a  class="logo" href="#"></a></li>
+                <li class="show-counter"></li>
+                <li><a href="#">Action</a></li>
+                <li><a href="#">Comedy</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <div class="main-container">
+        <div class="shows-container">
+            <ul class="display-list-of-shows"></ul>
+        </div>
```

Comment:
## Optional
- [ ] You have really done a great job team. The movies are not properly aligned, the gap on the left and right does not match, would you try to fine-tune them? This project can end up in the hands of a recruiter.